xonsh/style_tools.py

```diff
@@ -434,3 +434,18 @@ def norm_name(name):
     globals(),
     "DEFAULT_STYLE_DICT",
 )
+
+PTK2_STYLE = LazyObject(
```

Comment:
It doesn't make sense for this to be lazy if the this is just going to be copied first thing into the environment.

Reply:
as the default

Reply:
Right. Too much copy-pasting ;)

Reply:
No problem :)